internal/pkg/aws/ecs/ecs.go

```diff
@@ -99,9 +99,9 @@ func (e *ECS) Service(clusterName, serviceName string) (*Service, error) {
 	return nil, fmt.Errorf("cannot find service %s", serviceName)
 }
 
-// ServiceTasks calls ECS API and returns ECS tasks running by a service.
-func (e *ECS) ServiceTasks(cluster, service string) ([]*Task, error) {
-	return e.listTasks(cluster, withService(service))
+// ServiceRunningTasks calls ECS API and returns the ECS tasks spun up by the service, with the desired status to be set to be RUNNING.
+func (e *ECS) ServiceRunningTasks(cluster, service string) ([]*Task, error) {
+	return e.listTasks(cluster, withService(service), withRunningTasks())
```

Comment:
The sole purpose of adding `withRunningTasks()` is to maintain semantic consistency between the code and the function name. 

This does nothing to the result returned by the `listTasks` API call. This is because without `withRuningTasks()`, (i.e. without `DesiredStatus: RUNNING` in the `ListTasksInput`), the `DesiredStatus` is defaulted to `RUNNING` anyways.